### DISCUSSION TITLE
Is possible replicate data  between onpremise Kubernetes kafka cluster and Azure kuberenetes cluster?

### DISCUSSION THREAD

**Discussion by ohuarte-matrix** [strimzi/strimzi-kafka-operator#7497] [Q&A] [answered]
I need to deploy two strimzi kafka clusters, the first in on-premise kubernetes and the second in azure kubernetes

Is possible replicate data between on-premise  kafka and azure kafka ? I think it is possible, but i din't before

Many thanks for your help

**Comment by scholzj** [comment, marked as answer]
You cannot replicate the data in the Kafka's meaning of replication. You can mirror them between the two clusters using MirrorMaker (probably witht he newer Mirror Maker 2).

**Reply by ohuarte-matrix** [reply]
Many thanks!!
